spring-kotlin/aws-ssh-server-manager/src/main/kotlin/dev/santos/awssshservermanager/exception/UserExceptions.kt

```diff
@@ -0,0 +1,10 @@
+package dev.santos.awssshservermanager.exception
+
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.ResponseStatus
+
+@ResponseStatus(code = HttpStatus.CONFLICT)
+class DuplicateUserException(override val message: String) : RuntimeException()
```

Comment:
Do you really want to declare it as an [Unchecked Exception](https://docs.oracle.com/javase/tutorial/essential/exceptions/runtime.html)?